src/submission-handlers/testOrderHandler.ts

```diff
@@ -0,0 +1,61 @@
+import { map } from 'rxjs/operators';
+import { SubmissionHandler } from '..';
+import { OpenmrsEncounter, OHRIFormField } from '../api/types';
+import { EncounterContext } from '../ohri-form-context';
+
+export const TestOrderHandler: SubmissionHandler = {
+  handleFieldSubmission: (field: OHRIFormField, value: any, context: EncounterContext) => {
+    const constructNewOrder = {
+      action: 'new',
+      urgency: 'ROUTINE',
+      patient: context.patient?.id,
+      concept: value,
+      type: field?.questionOptions?.orderType,
+      careSetting: field?.questionOptions?.orderSettingUuid,
+      orderer: context?.encounterProvider,
+      encounter: context?.encounter?.uuid
+    };
+  
+    if (context.sessionMode === 'edit') {
+      const voidedPreviousOrders = context.encounter.orders?.map((element) => ({
```

Comment:
Why are you voiding all the encounter's orders? I thought we only void edited orders and link them to the new order?